pennylane/operation.py

```diff
@@ -398,6 +466,38 @@ def queue(self):
             QNode._current_context._append_op(self)
         return self  # so pre-constructed Expectation instances can be queued and returned in a single statement
 
+    def inverse_parameters(self, p):
+        """Function that acts on the parameters of the operation, inverting the operations action.
+
+        Args:
+            p (list): list of the operation parameters
+
+        Returns:
+            list: list containing the transformed parameters,
+            such that the operation implemented is now the inverse
+        """
+        if not p:
+            return p
+
+        ptmp = p[:]
+        if self.par_domain == 'A':
+            # should this be p[0].conj().T by default?
+            ptmp[0] = np.linalg.inv(p[0]) #pylint: disable=no-member
```

Comment:
This is subtle. Maybe we shouldn't have a default implementation for this until we have "upgraded" the par_domain to contain more information? Once `par_domain` contains more information about the matrix it gets easier to supply a good "default" operation for the inverse.

Reply:
That is true, I was definitely thinking about that when I left this comment. Since all existing built-in operations are unitary (and this likely won't be changed until after v0.2), I'm happy to keep this in as a stop-gap.

Reply:
Ok, fine.